Library/Formula/memo.rb

```diff
@@ -0,0 +1,16 @@
+class Memo < Formula
+  url "http://www.getmemo.org/memo-1.5.tar.gz"
+  homepage "http://www.getmemo.org"
+  sha1 "3e047b09e91d695f1767d1dd6d179732c07a5759"
+
+  def install
+    system "make"
```

Comment:
I just confirmed that it works.

What is `bin.mkpath` actually doing? Also, why is `man1.mkpath` not necessary?